agent/provider/src/market/negotiator.rs

```diff
@@ -16,7 +16,10 @@ pub enum ProposalResponse {
         offer: DemandOfferBase,
     },
     AcceptProposal,
-    RejectProposal,
+    #[display(fmt = "RejectProposal( reason: {:?})", reason)]
```

Comment:
It will be awfull message with Some()

Reply:
Let's cover it in other PR, as this is blocking others.

Reply:
reported as #776